econplayground/templates/main/cohort_password_form.html

```diff
@@ -0,0 +1,23 @@
+{% extends "base.html" %}
+{% load bootstrap4 %}
+
+{% block title %}Password Required{% endblock %}
+
+{% block content %}
+    <h1>Password Required</h1>
+    <p>
+        The instructor has set a password for this course.
+    </p>
+    <form class="form-group" method="post" action=".">{% csrf_token %}
+        <div class="form-group">
+            <input type="text" id="cohortPassword"
```

Comment:
It might make sense to use `type="password"` here

Reply:
Hmmmm, yeah. I'm using a text input for the password field in the instructor view for setting this password, as this password is really intended to be shared with students, so I thought it's fine for it to be visible there.

But from a UI standpoint, in this view.. I suppose `type="password"` would be what's expected. I'll make that change.